swan-lake/learn/using-the-cli-tools.md

```diff
@@ -126,27 +126,27 @@ These everyday commands are your best friends! They address the very basics of p
 <table class="cComandTable">
 <tr>
 <td class="cCommand">build</td>
-<td class="cDescription">Compile a Ballerina program, a single BAL file, a module, an entire project, or a single root module into an executable JAR file.
+<td class="cDescription">Compile a Ballerina program, a single BAL file, a module, an entire package, or a single root module into an executable JAR file. For more information, see <a href="/swan-lake/learn/running-ballerina-code">Running Ballerina Code</a>.
```

Comment:
We can't build modules. Also, root module is the package.
```suggestion
<td class="cDescription">Compile a Ballerina program, a single `.bal` file or an entire package into an executable JAR file. For more information, see <a href="/swan-lake/learn/running-ballerina-code">Running Ballerina Code</a>.
```

Reply:
Fixed.